### ISSUE TITLE
[REQUEST] Option to have Radarr/Sonarr notify Jellyseerr when item is imported.

### ISSUE THREAD

**Issue by devanteweary** [Fallenbagel/jellyseerr#960]
### Description

Hey there.

tldr; Have a webhook where Radarr/Sonarr will tell Jellyseerr when a movie/show was successfully imported.  

### Desired Behavior

Movie downloads, imports into Radarr/Sonarr, Webhook is sent to Jellyseerr to let it know the movie has been imported. 
This will show the movie as available and send notifications on the Jellyseerr side (even if it isn't quite imported into Plex quite yet).

### Additional Context

I thought about this because my users don't use JF, but I keep it around because I want them to be able to use my jfa-go invites + JF logins to seemlessly log into Jellyseerr. And OK maybe like 1 guy uses JF.

Anyway, I noticed JF recents scans were slower for whatever reason than Plex. I think it's because I have autoscan tell Plex immediately when a movie gets added so it's pretty instantaneous once the movie downloads.

Alrighty hope this makes sense!

### Code of Conduct

- [X] I agree to follow Jellyseerr's Code of Conduct

**Comment by Fallenbagel** [comment]
Jellyseerr already scans with sonarr/radarr. You just need to enable `enable scan` in services > radarr/sonarr

**Comment by devanteweary** [comment]
> Jellyseerr already scans with sonarr/radarr and Syncs availability. You just need to enable `enable scan` in services > radarr/sonarr
> 
> Also you can actually setup sonarr/radarr to immediately trigger a jellyfin scan when an item is imported through sonarr/radarr thus making the item be synced with jellyseerr in less than a minute or so of import (that's my workflow rn).

Hmmm interesting I thought so... but it doesn't seem to be working for me. 
I have "Scan" enabled in both Radarr and Sonarr services within Jellyseerr settings.

I checked Radarr/Sonarr connections and didn't see anywhere to tell it to notify Jellyseerr like you mentioned. How did you manage that? The only thing is stuff like Plex, Jellyfin, Notifarr, etc. But no Overseerr or Jellyseerr. Maybe I'm overlooking something.

Thanks for the reply, by the way!

p.s. I noticed it wasn't working because I had JF add media to library set to file creation date (and Radarr is dating them the date the movie came out). If I _don't_ turn this option on, it'll add every single upgrade to the "recents" list.

Well anyway what I think is happening is because the movie isn't being added to recents (cause of its date), Jellyseerr isn't seeing the movie on its Recents scan.

That's kinda why I thought well don't have it rely on Plex or JF but rather Radarr/Sonarr. (which you say is how you have it, though I can't see that option)

**Comment by gauthier-th** [comment]
> I checked Radarr/Sonarr connections and didn't see anywhere to tell it to notify Jellyseerr like you mentioned. How did you manage that? The only thing is stuff like Plex, Jellyfin, Notifarr, etc. But no Overseerr or Jellyseerr. Maybe I'm overlooking something.

Because there is no option to notify Jellyseerr. You can notify **Jellyfin** from Radarr/Sonarr, **not Jellyseerr**. Then as you said Jellyseerr will scan the items from "Recently Added" every 5min. If your movies/shows are not displayed in this Recently Added section, Jellyseerr will only scan them every 24h when the full Jellyfin library scan happens.

The only way of doing this using a webhook would be to send an API request to Jellyseerr, but Radarr does not let you specify any custom header (so you can specify the API key). I just opened an issue on Radarr asking whether it could be implemented: https://github.com/Radarr/Radarr/issues/10552

**Comment by gauthier-th** [comment]
Sorry, I got confused in my explanations.

We have theses jobs that updates the status of the requests:

- `Jellyfin Recently Added Scan` & `Jellyfin Full Library Scan`: runs every 5min and every 24h and get its data from the Recently Added section or from the whole library. These jobs are also used to get some info about the Jellyfin items.
- `Radarr Scan` / `Sonarr Scan`: scans the Radarr/Sonarr library every 24h.

So even if the Jellyfin library is still not up to date, the status of the request on Jellyseerr could still be accelerated with a webhook coming from Radarr/Sonarr (with a custom header for the API key).

**Comment by devanteweary** [comment]
> Sorry, I got confused in my explanations.
> 
> We have theses jobs that updates the status of the requests:
> 
> * `Jellyfin Recently Added Scan` & `Jellyfin Full Library Scan`: runs every 5min and every 24h and get its data from the Recently Added section or from the whole library. These jobs are also used to get some info about the Jellyfin items.
> * `Radarr Scan` / `Sonarr Scan`: scans the Radarr/Sonarr library every 24h.
> 
> So even if the Jellyfin library is still not up to date, the status of the request on Jellyseerr could still be accelerated with a webhook coming from Radarr/Sonarr (with a custom header for the API key).

Yes exactly what I was hoping to get added.  
It's mainly because it seems on average, the movie will be in Plex but Jellyseerr won't say available (and more importantly send out notifications) until maybe... 10 minutes or so later?

**Comment by Fallenbagel** [comment]
You can change the frequency of these jobs. Sonarr/radarr scan job will mark them available provided you have them on.
